cmd/crdvalidator/annotation/annotation.go

```diff
@@ -0,0 +1,7 @@
+package annotation
+
+const (
+	ValidationKey = "core.rukpak.io/safe-upgrade-validation"
+	Enabled       = "enabled"
```

Comment:
Apologies if this question has already been asked, but was there discussion about `"enabled"`/`"disabled"` vs `"true"`/`"false"` values for this annotation. I think `"true"`/`"false"` is what users would generally expect, but if there's a plan to have more than just `"enabled"`/`"disabled"` values, I could see an argument for non-boolean enums.

Reply:
https://github.com/operator-framework/rukpak/pull/268#discussion_r859147932 

There was some discussion about it here ^ That being said, I'm not sure if this is the field we would use to have different levels of validation like it was originally intended.

Reply:
I agree with you that it should be "true"/"false" as it more closely aligns with common standards. Added the change.